index.html

```diff
@@ -5,11 +5,161 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <title>VCC Conference</title>
 </head>
 
 <body>
-    <h1>My heading</h1>
+    <header>
+        <h1>vcc</h1>
+        <div class="nav-bar">
+            <input id="toggle" class="menu-toggle" type="checkbox" aria-hidden="true">
+            <label for="toggle" class="nav-icon" aria-hidden="true">
+                <span class="icon-line"></span>
+                <span class="icon-line"></span>
+                <span class="icon-line"></span>
+            </label>
+            <nav class="nav">
+                <ul class="nav-items">
+                    <li class="nav-item"><a href="#home">Home</a></li>
+                    <li class="nav-item"><a href="#speakers">Speakers</a></li>
+                    <li class="nav-item"><a href="#schedule">Schedule</a></li>
+                </ul>
+            </nav>
+        </div>
+        <section id="home" class="home">
+            <h1>visual communication conference</h1>
+            <p class=date>pasadena, california </p>
```

Comment:
Classnames should be wrapped in double quotes, like so:
```suggestion
            <p class="date">pasadena, california </p>
```